pkg/provider/helm/helm.go

```diff
@@ -390,12 +464,12 @@ func newUninstall(cfg *action.Configuration) *action.Uninstall {
 
```

Comment:
There's a reference to SPIRERepositoryName on L457, it needs to use the new field.

Reply:
Ah, yes - thanks. I had to feed it in as a parameter to the function so I've also included in the unit tests:

[6bc16cc](https://github.com/cofide/cofidectl/pull/278/commits/6bc16cc6ec742524903ff47e259a78dcd7e5ed62)